TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/DriveSubsystem.java

```diff
@@ -119,4 +119,12 @@ public void resetEncoders() {
     private double getHeading() {
         return imu.getRobotYawPitchRollAngles().getYaw(AngleUnit.DEGREES);
     }
+
+    public void changeSpeedMultiplier(){
+        if (speedMultiplier ==1){
+            speedMultiplier = 0.5;
+        }else{speedMultiplier=1;
+        }
+    }
```

Comment:
For the sake of readability, I prefer using the old method and specifying the values in Rotot.java. Also in line 126 you don't need the curly braces. So I think we will remove this method and replace it with the old one in the future.